#
#
# patch "libpurple/conversation.c"
#  from [cd6817f4863abafef969359f65cc025f9483b401]
#    to [19aac10ceebae1ee28788b6f0c02bcfcff0c8803]
#
============================================================
--- libpurple/conversation.c	cd6817f4863abafef969359f65cc025f9483b401
+++ libpurple/conversation.c	19aac10ceebae1ee28788b6f0c02bcfcff0c8803
@@ -1978,7 +1978,6 @@ purple_conv_chat_clear_users(PurpleConvC
 	}
 
 	g_hash_table_remove_all(chat->users);
-	chat->users = NULL;
 
 	g_list_free(users);
 	chat->in_room = NULL;
